### PR TITLE
Adds employee mode

### PR DESCRIPTION
**Pull request by jacqueline8711**
## Issue
This fixes the following issue(s): #1117 

- Hides lectures, grades and tuition fees for all people who are just employees.
- Adds setting to enable/disable employee mode for employees and employees that are also students.

- [x] make the change effective immediately without having to restart the app

### CODE REVIEW

File: app/src/main/java/de/tum/in/tumcampusapp/component/other/generic/drawer/DrawerMenuHelper.kt [hunk 1]
```diff
@@ -87,8 +92,8 @@ class DrawerMenuHelper(private val activity: Activity) {
                 SideNavigationItem(R.string.schedule, R.drawable.ic_outline_event_24px, CalendarActivity::class.java, true),
                 SideNavigationItem(R.string.my_lectures, R.drawable.ic_outline_school_24px, LecturesPersonalActivity::class.java, true),
```
Comment (pfent) [inline app/src/main/java/de/tum/in/tumcampusapp/component/other/generic/drawer/DrawerMenuHelper.kt]: Do employees have lectures?

Comment (jacqueline8711) [reply app/src/main/java/de/tum/in/tumcampusapp/component/other/generic/drawer/DrawerMenuHelper.kt]: Probably not

**Comment by pfent** [comment]
@jacqueline8711 looks good to me, is this still "Work in progress"?

**Comment by jacqueline8711** [comment]
@pfent I think that's it for now

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -94,6 +94,8 @@
     <string name="account" translatable="false">Account</string>
     <string name="logout">Log out</string>
     <string name="logout_message">Are you sure you want to log out?</string>
+    <string name="employee_mode">TUM Employee View Mode</string>
```
Comment (thellmund) [inline app/src/main/res/values/strings.xml]: Maybe that’s just me, but I find that this sounds a little odd. Maybe @kordianbruck or @pfent have an idea?

Comment (kordianbruck) [reply app/src/main/res/values/strings.xml]: > Employee Mode 

would really be enough.

Comment (pfent) [reply app/src/main/res/values/strings.xml]: I'd personally say "Non-Student Mode" :man_shrugging:

Comment (jacqueline8711) [reply app/src/main/res/values/strings.xml]: I like Employee Mode, it's short. What about the german label?

File: app/src/main/java/de/tum/in/tumcampusapp/component/other/generic/activity/BaseActivity.kt [hunk 3]
```diff
@@ -53,11 +54,22 @@ abstract class BaseActivity(private val layoutId: Int) : AppCompatActivity() {
...
         setContentView(layoutId)
-
+        previouslyEmployeeMode = Utils.getSettingBool(this, Const.EMPLOYEE_MODE, false)
         setUpToolbar()
         setUpDrawer()
     }
 
+    override fun onResume() {
+        super.onResume()
+        val newIsEmployeeMode = Utils.getSettingBool(this, Const.EMPLOYEE_MODE, false)
```
[the remark this answers is not in the document]
Comment (jacqueline8711) [reply app/src/main/java/de/tum/in/tumcampusapp/component/other/generic/activity/BaseActivity.kt]: Thanks for the suggestion :) This works well

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (kordianbruck) [reply app/src/main/res/values/strings.xml]: "Mitarbeitermodus"?

**Pull request merged**